### ISSUE TITLE
Header And Footer not visible

### ISSUE THREAD

**Issue by ujas3279** [sergiofraile/when_flutter_meets_tensorflow_part_5#1]
Please tell me how to fix header and footer proble.
![IMG-20200405-WA0011](https://user-images.githubusercontent.com/41516784/78504148-c86bff80-7788-11ea-8e7c-4ce09298f7ba.jpg)

**Comment by techyp** [comment]
Actually the code in this is not complete. The author have not completed the part V of this post(article on Medium), that is why your header and footer is not visible.

**Comment by techyp** [comment]
Also are you able to load the model??

**Comment by ujas3279** [comment]
yes i got accurate output in console only header and footer not visible.

**Comment by techyp** [comment]
Is it the same code from this repository or you modified the code.
Can you share the code here??

**Comment by techyp** [comment]
Also, my app is able to show the header and the footer as well, so maybe I could help you with that.
Only problem I am facing is that model fails to load.
So, if you can share the link

**Comment by techyp** [comment]
Hey, I've sent you a pull request completing the header and footer 2 or 3 days back.
